velox/exec/RowContainer.h

```diff
@@ -773,6 +773,18 @@ class RowContainer {
     return (row[rowColumn.nullByte()] & rowColumn.nullMask()) != 0;
   }
 
+  /// Returns true if the column at columnIndex in row is NaN.
+  template <
+      typename T,
+      std::enable_if_t<std::is_floating_point_v<T>, int32_t> = 0>
+  bool isNanAt(const char* row, int32_t columnIndex) {
+    const auto column = columnAt(columnIndex);
+    if (isNullAt(row, column.nullByte(), column.nullMask())) {
```

Comment:
In the most general sense its better to do the Null check here. 

Though in our case, null check happens earlier anyways. So we are guaranteed non-null values. Do you want to have an optimized version of this code that doesn't have the null check ?